includes/Ad_Manager.php

```diff
@@ -77,15 +77,21 @@ public function print_ad_manager_tag() {
 		if ( ! $enabled || ! $slot ) {
 			return;
 		}
+
+		$application_json = [];
+
+		$ad_attributes = [];
+		$ad_attributes[ 'type' ] = 'doubleclick';
+		$ad_attributes[ 'data-slot' ] = $slot;
+
+		$application_json['ad-attributes'] = $ad_attributes;
+		
+		$application_json = apply_filters( 'web_stories_ad_manager_json', $application_json );
```

Comment:
Would it be helpful to have `$slot` as the third argument?

Reply:
You can get it from the `$configuration` array.

Reply:
Not if a previous filter removes/overrides it. `$slot` as the third argument would give you easy access to the original, unmodified value.

Reply:
agree with @swissspidy will pass that variable as it should be unchanged